components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Engine/publishednodes.json

```diff
@@ -1,19 +1,25 @@
 [
   {
     "EndpointUrl": "opc.tcp://server1:49580",
+    "UseSecurity": false,
```

Comment:
Let's keep the test with previous version of `publishednodes.json` and add new test for the new version as all of those fields are optional and can be skipped. This way we can be sure that publisher can still consume older `publishednodes.json` files.

Reply:
Adding InlineData in test to check publishednodes.json with previous and new versions both, presuming that should cover what we need?

Reply:
InlineData data changes will be pushed in next commit. Please feel free to suggest otherwise.